integrations/deploy.yaml

```diff
@@ -89,16 +89,16 @@
     distro: 'rhel'
   quick_start: -1
 - <<: *linux
-  id: deploy-almalinux
+  id: deploy-rockylinux
   meta:
     <<: *linux_meta
-    name: Alma Linux
-    link: https://almalinux.org/
-    icon_filename: 'alma.png'
+    name: Rocky Linux
+    link: https://rockylinux.org/
+    icon_filename: 'rocky.svg'
```

Comment:
Just confirming, this is already in the repo we’re pulling images from?

Reply:
yes added it yesterday